src/main/java/io/debezium/connector/yugabytedb/consistent/Merger.java

```diff
@@ -64,10 +122,17 @@ private Optional<Message> peek() {
         return peeked;
     }
 
+    /**
+     * Get the front message from the queue to process further. This function also removes the
+     * message from the relevant merge slot and update the tablet safetime if required.
```

Comment:
nit: Can you update the java doc about updating the tablet safetime

Reply:
Done.